source/app/schedulers/instance_scheduler.py

```diff
@@ -119,6 +121,16 @@ def __init__(self, service, scheduler_configuration):
 
         self._usage_metrics = {"Started": {}, "Stopped": {}, "Resized": {}}
 
+    @property
+    def lambda_client(self):
+        """
+        Get the lambda client
+        :return: lambda client
+        """
+        if self._lambda_client is None:
+            self._lambda_client = get_client_with_standard_retry("lambda")
+        return self._lambda_client
+
```

Comment:
we seem to follow this lazy-load pattern a lot. Not a change for right now, but if we're going to keep using this pattern I would be tempted to create an @lazy_load decorator to improve readability